### DISCUSSION TITLE
使用 v2.1.4 版本无法选中文字

### DISCUSSION THREAD

**Discussion by reisojamin** [valor-x/hexo-theme-solitude#405] [General]
请确保阅读过本主题GitHub项目说明
  √  查询主题使用文档的相关内容和正确配置
  √  使用的是Solitude主题最新版本
  √  执行npm ls -dept 0检查是否已安装对应插件
  √   [FAQS](https://solitude-docs.efu.me/faq) 中无类似问题（网站挂了）

部署之后文章文字不能被选中，右键菜单开启后错位。
![Screenshot 2024-10-15 015807](https://github.com/user-attachments/assets/ee49d569-aa48-4ff1-94d4-144ce1de4472)
希望能解决一下，辛苦

**Comment by everfu** [comment]
应该解决了，更新下最新
